solution/ui/regulations/utilities/api.js

```diff
@@ -285,33 +252,6 @@ function httpApiPost(
     });
 }
 
-// eslint-disable-next-line no-unused-vars
-function httpApiPut(urlPath, { data, params } = {}, cacheResponse = true) {
-    return fetchJson({
-        url: `${config.apiPath}/${urlPath}`,
-        options: {
-            method: "PUT",
-            headers: authHeader(token),
-            params,
-            body: JSON.stringify(data),
-        },
-        cacheResponse,
-    });
-}
-
-// eslint-disable-next-line no-unused-vars
-function httpApiDelete(urlPath, { data, params } = {}, cacheResponse = true) {
-    return fetchJson({
-        url: `${config.apiPath}/${urlPath}`,
-        options: {
-            method: "DELETE",
-            headers: authHeader(token),
-            params,
-            body: JSON.stringify(data),
-        },
-        cacheResponse,
-    });
-}
```

Comment:
more cleanup of methods that we're not using and as far as I know have no intention of using.